handbook/marketing/product-marketing/product-roadmap.md

```diff
@@ -0,0 +1,22 @@
+# Product roadmap
+
+## What is this product roadmap?
+
+In September 2021, the Sourcegraph Product and Product Marketing teams collaborated on building a roadmap deck communicating rolling releases in a 6-month outlook. [This resource is available here](https://docs.google.com/presentation/d/1o3R8WUIhzzRz0x5laTwVcizOzVWrMBe5MCAz74H45Ss/edit?usp=sharing) and covers the new product releases that are coming in the foreseeable future, as well as the value each provides to Sourcegaph customers. **In essence, this roadmap provides a higher-level perspective for customers on the big problems we are solving and why and as such, it potentially doesn't cover every specific release we are working on**.
```

Comment:
Is the link to the roadmap deck private? If so, do we need to make it clear who can access it?

Reply:
Private - clarified and added the public product direction pages as well for good balance.